internal/cryptutil/hmac.go

```diff
@@ -7,6 +7,11 @@ import (
 	"time"
 )
 
+const (
+	// DefaultLeeway defines the default leeway for matching NotBefore/Expiry claims.
+	DefaultLeeway = 1.0 * time.Minute
```

Comment:
Maybe lean on the kerberos default of 5m?

Reply:
I've gone back and forth about this. Just for some context, I wanted to make this value consistent with how we handle JWTs.  The JWT [spec](https://tools.ietf.org/html/rfc7519#section-4.1.4)  says the intent of leeway for jwt based time claims is to account for clock skew.  I noticed that our [JWT](https://godoc.org/github.com/square/go-jose/jwt#pkg-constants) library uses one minute leeway and went with that. 

> Implementers MAY provide for some small leeway, usually no more than a few minutes, to account for clock skew.

 I don't know much about why Kerberos's default leeway but I think it's within a reasonable time-frame. AD seems to use five minutes [as well](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/maximum-tolerance-for-computer-clock-synchronization). 

My intuition tells me that five minutes is quite a long time when we are looking at probably ~10-15 minute route tokens to begin with, effectively adding anywhere from 1/3-1/2 duration on either end of a token's validity. 

I think five minutes makes sense, but think we could be more aggressive as I think it's less likely (...one would hope?) that server machines would have as "drifty" time skew as desktop machines. I'd love to know what folks like [aws](https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html) use for their allowed leeway, basically.

Reply:
5 is definitely a hold over from when you couldn't count on fantastic clock sync.  Since this is server to server (right?) 1m could be reasonable.

Reply:
> sync. Since this is server to server (right?)

Yes

> 1m could be reasonable

Let's see how it goes 👍